ethereum/contract_addresses.go

```diff
@@ -103,7 +103,7 @@ var ChainIDToContractAddresses = map[int]ContractAddresses{
 		Exchange:            common.HexToAddress("0x30589010550762d2f0d06f650d8e8b6ade6dbf4b"),
 		Coordinator:         common.HexToAddress("0x2ba02e03ee0029311e0f43715307870a3e701b53"),
 		CoordinatorRegistry: common.HexToAddress("0x09fb99968c016a3ff537bf58fb3d9fe55a7975d5"),
-		DevUtils:            common.HexToAddress("0x1e3616bc5144362f95d72de41874395567697e93"),
+		DevUtils:            common.HexToAddress("0x80101f90446d9e19a80109ac2767c8c957debb29"),
```

Comment:
Did we meant to do this? Judging by our cheat sheet [this address is a version of DevUtils is designed for protocol v3](https://0x.org/docs/guides/0x-cheat-sheet).

Reply:
This was an oversight. Fixing.